crawl-ref/source/dat/des/portals/bailey.des

```diff
@@ -450,16 +450,17 @@ ENDMAP
 #
 # Apart from weapons and armours left by the monsters, there are about 10
 # potions useful in battle. If those monsters drink them first, of course...
-# Plus, there is always one stack of very good potions.
 NAME:   bailey_axe_2
 ORIENT: encompass
 TAGS:   bailey_axe no_rotate
 : axe_returning(_G)
```

Comment:
the swarm of boomers doesn't really work here, and also baileys give out too many boomers for modern crawl standards.